### PR TITLE
Highlighted - NavBar and Category

### PR DESCRIPTION
**Pull request by linh224**
I updated Navbar, so we can know current tab that we are watching.
The item of the category also is highlighted when we click on every single item in category.
Please take a look and comment!
Thank you :)

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -42,3 +42,10 @@
     transform: rotate(360deg);
   }
 }
+.clicked-div{
+  background-color: rgb(197, 235, 114);
```
Comment (sieday) [inline src/App.css:46]: I will update to match color scheme

File: src/NavBar.jsx [hunk 2]
```diff
@@ -15,31 +18,51 @@ const NavBar = () => {
...
+                }
+              >
+                About
+              </NavLink>
+            </li>
+            <li className="nav-item">
+              <NavLink
+                to="/contact"
+                className={({ isActive }) =>
+                  `linkStyle ${isActive ? activeLink : normalLink}`
```
Comment (sieday) [inline src/NavBar.jsx:58]: Smart. Works perfect, love it

File: src/components/About.jsx [hunk 3]
```diff
@@ -21,7 +21,8 @@ function About() {
           <div className="">
             <span className="team p-1 fs-2 fw-bold ">API Used</span>
             <ul className="custom-list fs-6 fw-bold p-3">
-              <li>NewYork API</li>
+              <li>NewYorkTimes API</li>
```
Comment (sieday) [inline src/components/About.jsx:24]: This is cool, do we have any references we should also add here? Not sure if we used any additional resources enough to reasonably document them.

**Review by sieday — APPROVED** [review]
Looks perfect, I am cool to merge this in. Thanks Linh!

File: src/components/About.jsx (hunk 3 above)
Comment (deanajackson) [reply src/components/About.jsx:24]: This should be Google Books API and New York Times Bestsellers API, but I can change this later.

**Pull request merged**